extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -116,16 +148,43 @@
   background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="%23fff" viewBox="0 0 18 18"><path d="M15 8.25H5.87l4.19-4.19L9 3 3 9l6 6 1.06-1.06-4.19-4.19H15v-1.5z" /></svg>');
 }
 
-.amp-lightbox-viewer-button-close {
+.amp-lightbox-viewer-button-close,
+.amp-lightbox-viewer-button-gallery,
+.amp-lightbox-viewer-button-back {
   top: 0;
```

Comment:
remove this selector and add top in individual selectors below - that way you will save up more kbs :) and the CSS would be better readable